### DISCUSSION TITLE
enable tab complete for mod commands

### DISCUSSION THREAD

**Discussion by lerai** [EssentialsX/Essentials#5188] [Q&A] [answered]
I'm setting up a server with lots of mods. One of the main ones is pixelmon (along with its command addons. example: pixelextras), as for allowing the use of specific commands it is not a problem. The point is to be able to use the TAB to facilitate the use of the commands. The problem is that TAB doesn't work for those. How could I activate this?

**Comment by Chew** [comment, marked as answer]
\>Pixelmon<br>
\>mods

This implies some form of Bukkit+Forge hybrid, which isn't supported. If this suggestion also applies to a normal server, please let us know.
